server/reports.go

```diff
@@ -128,7 +128,7 @@ func fillInGeneratedReports(clusterNames []types.ClusterName, reports map[types.
 	}
 
 	// it must be ok now
-	generatedReports.Status = "OK"
+	generatedReports.Status = "ok"
```

Comment:
probably worth to introduce new constant there?

Reply:
just for this occurence or for all the "ok" status in the code?

Reply:
yes, and exported `const` is reachable in tests as well.